packages/firestore/src/core/firestore_client.ts

```diff
@@ -573,6 +581,29 @@ export class FirestoreClient {
     });
     return deferred.promise;
   }
+
+  loadBundle(
+    data: ReadableStream<Uint8Array> | ArrayBuffer | string,
+    resultTask: LoadBundleTask
+  ): void {
+    this.verifyNotTerminated();
+
+    const reader = createBundleReader(
+      data,
+      newSerializer(this.databaseInfo.databaseId)
+    );
+    this.asyncQueue.enqueueAndForget(async () => {
+      loadBundle(this.syncEngine, reader, resultTask);
+      return resultTask.catch(e => {
+        logWarn(LOG_TAG, `Loading bundle failed with ${e}`);
+      });
```

Comment:
Same comment as above.

Reply:
This looks different now, but should be fixed.